code/modules/reagents/reagent_containers/patch.dm

```diff
@@ -13,6 +13,16 @@
 	..()
 	icon_state = "bandaid" // thanks inheritance
 
+/obj/item/weapon/reagent_containers/pill/patch/attack(mob/living/M, mob/user)
+	if(!canconsume(M, user))
```

Comment:
Canconsume checks if your mouth is clear, that's all

Just remove the call from the proc